docs/docs_skyrim_platform.md

```diff
@@ -353,6 +355,39 @@ browser.executeJavaScript("console.log('Hello CEF')");
 const str = browser.getToken();
 ```
 
+#### Talking back to the game
+
+Use `window.skyrimPlatform.sendMessage` on the browser side to talk back to the game. `sendMessage` accepts zero or more JSON-serializable values.
+```js
+window.skyrimPlatform.sendMessage({ foo: 'bar' });
+window.skyrimPlatform.sendMessage(1, 2, 3, "yay");
+window.skyrimPlatform.sendMessage();
+```
+
+You can call this function whenever you want: as a button callback, etc.
+```html
+<input type="button" value="Click me" onclick="window.skyrimPlatform.sendMessage({ foo: 'bar' });">
+```
+
+Calls to `sendMessage` result in a `browserMessage` event on the SP side. You can handle these events as any others.
+```ts
+on("browserMessage", (event) => {
+  printConsole(JSON.stringify(event.arguments));
```

Comment:
Is it game console (which opens with `)? BTW maybe add log level here to be future-proof? Probably optional, with INFO by default

Reply:
Yes. Good idea. But it's out of PR's scope